docs/cost_estimates.md

```diff
@@ -0,0 +1,98 @@
+| Cloud Services Administrators | Manage and monitor cloud-based resources and services. |
+| Security Analysts | Monitor for security breaches, conduct assessments. |
+| Compliance Officers | Ensure compliance with regulations, manage audits. |
+| Technical Trainers | Deliver ongoing training on software updates and features. |
+| Project Managers (for ongoing projects) | Coordinate projects related to the software. |
+| Risk Managers | Monitor and assess risks, develop mitigation strategies. |
+
+### Backstopping
+
+The backstopping structure is designed to provide a support system for both key roles and other experts involved in the project. It ensures that specialized knowledge and leadership are always available, minimizing the impact of any individual's temporary absence. This structure is particularly important for maintaining progress and quality in projects that require a high level of expertise and coordination. The actual scope of the backstopping structure depends on the specific project, its complexity and critical elements, and will be specified in the terms of reference for the specific project.
+
+## Estimate Run Cost
+
+| Cost Category | Description | Estimated Cost (€) |
+|---------------|-------------|-------------------|
+| Infrastructure Costs *) | - Hardware (servers, networking equipment)<br>- Cloud services (compute, storage, networking)<br>- Data center costs (space, power, cooling) | € |
+| Personnel Costs | - Salaries and benefits for IT support staff, system/network administrators, software developers, DBAs, security analysts, etc.<br>- Training and professional development (see table on job roles in run phase)| € |
+| Software Support and Maintenance | - External support contracts (if applicable)<br>- Internal resources dedicated to updates, patches, and troubleshooting<br>- Open-source community contributions or membership fees (if applicable) | € |
+| Software Development and Customization | - Ongoing development for custom features and integrations<br>- Bug fixes and performance enhancements | € |
+| Security and Compliance | - Security tools and services (firewalls, intrusion detection systems)<br>- Compliance audits and certifications (e.g., GDPR)<br>- Security training for staff | € |
+| Backup and Disaster Recovery | - Backup software and hardware<br>- Cloud storage for backups<br>- Disaster recovery planning and implementation | € |
+| Networking and Connectivity | - Internet bandwidth<br>- VPN licenses for remote access<br>- Network monitoring tools | € |
+| Monitoring and Management Tools | - System monitoring (performance, health)<br>- Application performance monitoring (APM) tools<br>- Log management and analysis tools | € |
+| Cloud Services Fees | - Fees for cloud computing resources<br>- Data transfer fees<br>- API request fees | € |
+| Licenses for Non-OSS Components | - Licensing fees for any non-open-source software or components used in conjunction with the open-source application | € |
```

Comment:
this contradicts a bit the beginning of the chapter that license costs are needed regardless of open source